examples/common/builders.py

```diff
@@ -17,8 +17,9 @@
                                       LinearWithWarmupScheduler)
 
 from examples.common.fdiff import FDiffMetrics
-from examples.common.optim.lion import DecoupledLionW
+from examples.common.optim import DecoupledLionW, ClipLion, AdaLRLion
```

Comment:
Should we also call these "decoupled"
```suggestion
from examples.common.optim import DecoupledLionW, DecoupledClipLion, DecoupledAdaLRLion
```